themes/scss/map/dashboard-info.scss

```diff
@@ -73,7 +70,8 @@
   margin: 0 auto;
   border: 2px solid #FFF;
   border-radius: 50%;
-  background: url('../img/dashboard-show.svg') no-repeat center center;
+  color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`